app_template_iframe/assets/iframe.html

```diff
@@ -1,16 +1,19 @@
+<!doctype html>
 <html>
 <head>
   <meta charset="utf-8">
-  <!-- http://garden.zendesk.com -->
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@zendeskgarden/css-bedrock@7.0" type="text/css">
+  <!--   See "Using Zendesk Garden":
+    https://developer.zendesk.com/apps/docs/developer-guide/setup#using-zendesk-garden
+    https://garden.zendesk.com/css-components/utilities/typography/
+   -->
+ <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@zendeskgarden/css-utilities@4/dist/index.min.css">
```

Comment:
Any reason you removed bedrock from the stylesheet?

Reply:
It resets all default stylings, including browser ones I  believe, which I was thinking for a starting point for an app might be kind of jarring.

Reply:
@ocke +1 then and a merge? The default template using 'u-gamma' is incorrect now as there is no 'u-gamma' in Garden anymore. Bringing in 'Utilities' was to do something similar as before on the H2 block.

Reply:
@bryan-flynn-zd I feel we should at least have css resets (normalize.css) in there. Bedrock helps inject [normalize.css](https://github.com/necolas/normalize.css/), default zendesk prefered fonts etc.

Reply:
Thanks @sarmadsangi for that additional info. I added back Bedrock. I also included both libraries separately with Bedrock first -- I didn't want Bedrock to be loaded after Utilities in one JSDelivr blob. I also made the links to be version specific, just to avoid any possible inconsistencies between versions.